### PR TITLE
Improvement: small gui changes

### PR DESCRIPTION
**Pull request by DeDiamondPro**
## Description

Various small changes to gui
- [x] back/forward carrots
- [x] update to new logo
- [X] remove background of bottom buttons
- [X] better dropshadow
- [X] revised animations

todo in next PR:
- [ ] update sidebar
- [ ] update sidebar gray color
- [ ] slider pressed state/cursors
- [ ] new color selector (nxtday)

## Checklist
- [X] I made a clear description of what was changed
- [X] I stated why these changes were necessary
- [X] I updated documentation or said what needs to be updated
- [X] I made sure these changes are backwards compatible
- [X] This pull request is for one feature/bug fix

### CODE REVIEW

**Review by Wyvest — APPROVED** [review]
thumbs up emoji

**Comment by CaledonianEH** [comment]
**Issues found**
- Credits page needs some work. The logo needs to be fixed, and libraries need to be aligned with the developer team. See figure 1

**Issues fixed**
- Bottom button backgrounds removed

If these issues are only on my end, then please let me know and I will see where my issues come from

**Figure 1**
wanCahn

**Comment by Wyvest** [comment]
> **Issues found**
> 
> * Credits page needs some work. The logo needs to be fixed, and libraries need to be aligned with the developer team. See figure 1
> 
> **Issues fixed**
> 
> * Bottom button backgrounds removed
> 
> If these issues are only on my end, then please let me know and I will see where my issues come from
> 
> **Figure 1** wanCahn

that doesn't happen on my computer lmao

**Comment by CaledonianEH** [comment]
**PASSES QC**

**Review by CaledonianEH — APPROVED** [review]
👍

File: src/main/java/cc/polyfrost/oneconfig/gui/animations/EaseOutBump.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+     * @param reverse  Reverse the animation
+     */
+    public EaseOutBump(int duration, float start, float end, boolean reverse) {
+        super(duration, start, end, reverse);
+    }
+
+    // Courtesy of https://easings.net/
+    @Override
+    protected float animate(float x) {
+        double constant1 = 1.7;
```
Comment (nxtdaydelivery) [inline src/main/java/cc/polyfrost/oneconfig/gui/animations/EaseOutBump.java:18]: can you replace these with private static final ones?

**Review by nxtdaydelivery — COMMENTED** [review]
see comment

**Comment by nxtdaydelivery** [comment]
good looks to me

**Review by nxtdaydelivery — APPROVED** [review]
good looks to me

**Review by Wyvest — APPROVED** [review]

**Pull request merged**